iam/api-client/service_accounts_test.py

```diff
@@ -13,26 +13,33 @@
 # limitations under the License.
 
 import os
-import random
+import uuid
+
+from googleapiclient.errors import HttpError
 
 import service_accounts
 
 
 def test_service_accounts(capsys):
     project_id = os.environ['GCLOUD_PROJECT']
-    rand = str(random.randint(0, 1000))
-    name = 'python-test-' + rand
+    name = 'python-test-{}'.format(str(uuid.uuid4()).split('-')[0])
```

Comment:
I feel like machine ID would be ideal - are you just using UUID4 for a random value?

Reply:
Yes. Due to the length limitation, I needed to use only the first part, but it's still far better than `random.randint(0, 1000)`.